src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Windows.props

```diff
@@ -0,0 +1,16 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Windows.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Condition=" '$(DisableImplicitFrameworkReferences)' != 'true' and '$(TargetFrameworkIdentifier)' == '.NETCoreApp' and '$(_TargetFrameworkVersionWithoutV)' &gt;= '5.0' and '$(TargetPlatformIdentifier)' == 'Windows' and '$(TargetPlatformVersion)' &gt;= '10.0'">
```

Comment:
Use MSBuild intrinsic version comparison functions for these version comparisons.

Also it would be good to apply the [workload .props pattern](https://github.com/dotnet/designs/blob/master/accepted/2020/workloads/workload-resolvers.md#workload-props-files) here, where we have a .targets file that's conditionally imported based on the TargetPlatformIdentifier, and it sets a single internal property which controls whether the FrameworkReference is included in the props file.  That doesn't necessarily need to be done in this PR though.